src/python_testing/matter_testing_infrastructure/chip/testing/commissioning.py

```diff
@@ -0,0 +1,137 @@
+#
+#    Copyright (c) 2022-2025 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+import logging
+from dataclasses import dataclass
+from typing import List, Optional
+
+from chip import ChipDeviceCtrl  # Needed before chip.FabricAdmin
```

Comment:
we do not import chip.FabricAdmin here anymore. Do we need this import now in some specific order or at all? Comment at least is not relevant anymore.